app/code/community/Bolt/Boltpay/Model/Quote.php

```diff
@@ -175,24 +175,33 @@ function ($item) use ($quote, &$calculatedTotal, $boltHelper) {
             /////////////////////////////////////////////////////////////////////////////////////////
         } else {
 
-            // Billing / shipping address fields that are required when the address data is sent to Bolt.
-            $requiredAddressFields = array(
+            $billingAddress  = $quote->getBillingAddress();
+            $shippingAddress = $quote->getShippingAddress();
+
+            /*////////////////////////////////////////////////////////////////////////////
+               -------------------------------------
+                Billing / shipping address fields that are required
+                when the address data is sent to Bolt.
+                ------------------------------------
                 'first_name',
                 'last_name',
                 'street_address1',
                 'locality',
                 'region',
```

Comment:
I don't think region is required depending on country

Reply:
@daisy1754 The doc https://docs.bolt.com/reference#create-order needs to be updated to express this.  It simply list region as a required field.

I certainly have seen that for some countries, like the Bahamas, it is not required by the Bolt full checkout modal.  Shall I update the doc?